scene/main/viewport.cpp

```diff
@@ -1955,6 +1957,31 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 				gui.drag_data=Variant(); //always clear
 			}*/
 
+			// in case the mouse was released after for example dragging a scrollbar,
```

Comment:
Nitpick, but start comment with an uppercase letter :)

Reply:
There are a lot of existing (full-sentence, not single-word) comments with varying formatting too.